Mobile desktop

ragangithub opened the pull request:
# In this branch
- Added HTML file
- Added CSS file
- Added JS file
- Updated README file
- Link to Loom: https://www.loom.com/share/ad2554f7fd8b4e9eb9432cd65807c246

ragangithub committed:
finish mobile and desktop

ragangithub committed:
Add about page

ragangithub committed:
Add JS file

ragangithub committed:
Add menu for mobile

ragangithub committed:
Fix linter error

ragangithub committed:
Update README

ragangithub committed:
Update JS file

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,152 @@
+          />
+        </div>
+      </div>
+    </section>
```
- [ ] Please change the horizontal line below the title to be shorter and has the same color used in the template.
- [ ] Please make sure that this section matches the given design (on the mobile version as well). This is what it looks like currently on the live demo:

![image](https://user-images.githubusercontent.com/62290545/233100261-785a5d32-6912-432f-b7a6-ced1227fe6fd.png)

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,207 @@
+          <li>Logoout</li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logo">
+          <img src="./images/logo.png" alt="Empowering African women log" />
+        </div>
+        <ul class="menu-links desktop">
+          <i class="close fa fa-times"></i>
+          <li><a class="nav-link" href="./index.html">HOME</a></li>
+          <li><a class="nav-link" href="./about.html">ABOUT</a></li>
+          <li><a class="nav-link" href="">JOIN</a></li>
+          <li><a class="nav-link" href="">SPONSOR</a></li>
+          <li><a class="nav-link" href="">NEWS</a></li>
+          <li><a class="campaign nav-link" href="">CAMPAIGN</a></li>
+        </ul>
+      </nav>
```
- [ ] Kindly make sure that the nav links stay on the right side of the navbar to match the template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233100712-51f7b8b8-e132-4990-89ba-4564ffffbcbb.png) | ![image](https://user-images.githubusercontent.com/62290545/233100917-3fe8c8b5-3a3f-4b4c-8aad-c68b46e1ff31.png) |

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        <nav>
+          <i class="fa fa-bars menu"></i>
+        </nav>
+        <span>"HELLO! AFRICAN WOMEN"</span>
+        <h1>
+          EMPOWERING AFRICAN WOMEN <br />
+          GLOBAL SUMMIT <br />
+          2023
+        </h1>
+        <p>
+          A joyful celebration believing in the value of empowering women to
+          achieve their destiny by removing barriers that hold them back with
+          people from over 54 countries is taking place in April, Ethiopia.
+        </p>
+        <div class="headerDate">2023.05.20(THU) ~ 21(FRI)</div>
```
- [ ] Please increase the size of the header text to match the design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233101867-fec4be67-b4c3-4549-8042-d042f8e43f33.png) | ![image](https://user-images.githubusercontent.com/62290545/233101964-286ba1e1-05ec-4670-8bf1-6da0f5ad7e34.png) |

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,152 @@
+      </div>
+    </section>
+
+    <section class="empowering">
+      <h2>EMPOWERING AFRICAN WOMEN <br />conference discussion issues</h2>
+      <hr class="empowering-hr" />
+
+      <ul class="lists">
+        <li class="list">
+          1. Place women as leaders and give them decision making roles
+        </li>
+        <li class="list">2. More Job Opportunities for Women</li>
+        <li class="list">
+          3. Invest in Women’s Entrepreneurial Ideas, Emotionally and
+          Financially
+        </li>
+        <li class="list">4. Taking Action against Unpaid Labour Work</li>
+        <li class="list">5. Mentoring Women Professionally and Personally</li>
+      </ul>
+    </section>
```
- [ ] Kindly edit this section to be more like the original design. As the requirements indicated, you can change the contents of the page but you'll have to stick to the design guidelines (colors, positioning, font style, ...etc) of the original design. Please add a paragraph and then an image below it to match the design.
- [ ] Please change the horizontal line below the title to be shorter and has the same color used in the template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233102891-d424909f-bc60-406b-ab59-0a1ddc27b992.png) | ![image](https://user-images.githubusercontent.com/62290545/233102772-441a09d6-ea43-419f-a655-38034ecb551b.png) |

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        © 2023 International Conference ,African Women Empowerment <br />Some
+        Rights Reserved.
+      </p>
+    </footer>
```
- [ ] Please remove the footer section in the mobile view of the index page to match the original template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233104863-b9886c2a-89bd-4ded-9136-df3a2c9f8a32.png) | ![image](https://user-images.githubusercontent.com/62290545/233104915-b6e49519-32d7-4e7c-b9e2-116d0b35acb3.png) |

- [ ] Please rerecord the captured video after you're done with these changes, as the design will change quite a bit from what was first recorded.

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,207 @@
+      <h3>Speakears</h3>
+      <hr />
+      <ul class="container"></ul>
+      <div class="more">MORE<i class="fa-solid fa-chevron-down"></i></div>
```
- [ ] For semantic purposes, kindly use a `button` tag here instead of using a `div` which functions as a button.

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,152 @@
+      </div>
+    </section>
+
+    <footer class="about-footer">
+      <div>
+        <img
+          class="footer-image"
+          src="./images/logo.png"
+          alt="Empowering African women log"
+        />
+      </div>
+      <p class="footer-detail">
+        © 2022 International Conference ,African Women Empowerment <br />Some
+        Rights Reserved.
+      </p>
+    </footer>
```
- [ ] Please add the "Partner" section to the mobile version of the "About" page to match the given design.
- [ ] Kindly make sure that the footer section in the "About" page has a different background color on the mobile view than the one you used.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/233106248-035bda5d-86fe-4a62-b4b5-0fab44b65f72.png) | ![image](https://user-images.githubusercontent.com/62290545/233106171-cbe1b908-c1a5-4198-8bf3-d4bc6bd13e8a.png) |

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        <div class="logo">
+          <img src="./images/logo.png" alt="Empowering African women log" />
+        </div>
+        <ul class="menu-links desktop">
+          <i class="close fa fa-times"></i>
+          <li><a class="nav-link" href="./index.html">HOME</a></li>
+          <li><a class="nav-link" href="./about.html">ABOUT</a></li>
+          <li><a class="nav-link" href="">JOIN</a></li>
+          <li><a class="nav-link" href="">SPONSOR</a></li>
+          <li><a class="nav-link" href="">NEWS</a></li>
+          <li><a class="campaign nav-link" href="">CAMPAIGN</a></li>
+        </ul>
```
- [ ] [OPTIONAL] I'd suggest you make the navbar `fixed` position on the mobile view to help the user access the nav menu if they are scrolling through the page. (IE: if the page is scrolled down, you can't access the nav menu)

ShadyShawkat reviewed (changes requested):
## Hi @ragangithub, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

![Image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub committed:
Update HTML and CSS file

ragangithub commented:
- Changed the horizontal line, made the images visible, and corrected the sections to have a similar design to the original 
design in 'about page'
- made "Nav links" stay on the right side
- Increased the size of header text
-  Removed "partner section" and footer from the index page
- Used button instead of div
- Added "partner section" to the mobile version of the 'About page'
- Changed the background color of the footer "About page" on the mobile version
- Link to new loom video:https://www.loom.com/share/22ff84fac56f461d8b655cde932e6777

ichala commented on about.html:
```diff
@@ -0,0 +1,189 @@
+      </div>
+    </section>
+
+    <section class="empowering">
+      <h2>EMPOWERING AFRICAN WOMEN <br />conference discussion issues</h2>
+      <hr class="empowering-hr" />
+
+      <div class="issues">
+        <p>
+          Place women as leaders and give them decision making roles,More Job
+          Opportunities for Women,Invest in Women’s Entrepreneurial Ideas,
+          Emotionally and Financially, Taking Action against Unpaid Labour Work
+          and,Mentoring Women Professionally and Personally
+        </p>
+        <div class="pan-african">
+          <img src="./images/Africa-women's-organisation-log.jpg" alt="" />
+        </div>
+      </div>
+    </section>
```
- [ ] as men,tioned by the previous reviewer and to match figma design kindly make sure the text below the title is centred 🙏 

![image](https://user-images.githubusercontent.com/89282221/233166947-b91187d5-b3f8-49d4-9f65-0f06cac644c7.png)

ichala reviewed (changes requested):
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/28307960/157369993-50d5ba12-099f-4f2a-9a19-5e57ee2d1c96.gif)

## To Highlight 🎉

- Almost all changes made✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub committed:
Fix CSS issues

ragangithub commented:
- Made the text below the title align in the center

arslanbisharat commented:
Hi @ragangithub,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
